modules/java/partials/directory-diagrams.adoc

```diff
@@ -0,0 +1,18 @@
+// This file contains diagrams for the installation directory structure
+
+// # tag::couchbase-lite-zip[]
+[source, text]
```

Comment:
I think you have an outdated version of the zip file, here.

As of 2.8.6, we no longer support CentOS6.  The support directory contains exactly two things: the libc++ directory and the libicu directory